### ISSUE TITLE
The save button is missing the appropriate CSS styling after language is selected

### ISSUE THREAD

**Issue by KrolPolski** [Welhox/Transcendence#56]
The language preference save button is missing the appropriate CSS styling and I will need to do some more accessibility testing 
to ensure that after multiple language support has been added that the settings page remains accessible.
